source/frontend/src/components/maps/leaflet/LayerPopup/LayerPopupView.test.tsx

```diff
@@ -227,6 +227,111 @@ describe('LayerPopupView component', () => {
+
+    it('only shows research file option if file disposed or retired', async () => {
+      const { getByTestId, getByText, queryByText } = setup({
+        layerPopup: {
+          latlng: undefined,
+          layers: [],
+        },
+        featureDataset: {
+          pimsFeature: {
+            type: 'Feature',
+            properties: {
+              ...EmptyPropertyLocation,
+              IS_RETIRED: true,
+              IS_DISPOSED: true,
+              PROPERTY_ID: 1,
+            },
+            geometry: { type: 'Point', coordinates: [] },
+          },
+          location: { lat: 0, lng: 0 },
+          fileLocation: null,
+          parcelFeature: null,
+          regionFeature: null,
+          districtFeature: null,
+          municipalityFeature: null,
+          highwayFeature: null,
+          selectingComponentId: null,
+          crownLandLeasesFeature: null,
+          crownLandLicensesFeature: null,
+          crownLandTenuresFeature: null,
+          crownLandInventoryFeature: null,
+          crownLandInclusionsFeature: null,
+        },
+
+        claims: [Claims.RESEARCH_ADD],
+      });
+      const ellipsis = getByTestId('fly-out-ellipsis');
+      await act(async () => userEvent.click(ellipsis));
+      const link = getByText('Research File');
+      expect(queryByText('Acquisition File')).toBeNull();
+      expect(queryByText('Disposition File')).toBeNull();
+      expect(queryByText('Lease/Licence File')).toBeNull();
+      expect(link).toBeVisible();
+    });
+
+    it('only shows all file options if file not disposed or retired', async () => {
+      const { getByTestId, getByText, queryByText } = setup({
+        layerPopup: {
+          latlng: undefined,
+          layers: [],
+        },
+        featureDataset: {
+          pimsFeature: {
+            type: 'Feature',
+            properties: {
+              ...EmptyPropertyLocation,
+              IS_RETIRED: false,
+              IS_DISPOSED: false,
+              PROPERTY_ID: 1,
+            },
+            geometry: { type: 'Point', coordinates: [] },
+          },
+          location: { lat: 0, lng: 0 },
+          fileLocation: null,
+          parcelFeature: null,
+          regionFeature: null,
+          districtFeature: null,
+          municipalityFeature: null,
+          highwayFeature: null,
+          selectingComponentId: null,
+          crownLandLeasesFeature: null,
+          crownLandLicensesFeature: null,
+          crownLandTenuresFeature: null,
+          crownLandInventoryFeature: null,
+          crownLandInclusionsFeature: null,
+        },
+
+        claims: [Claims.RESEARCH_ADD],
+      });
+      const ellipsis = getByTestId('fly-out-ellipsis');
+      await act(async () => userEvent.click(ellipsis));
+      const link = getByText('Research File');
+      expect(queryByText('Acquisition File')).toBeNull();
+      expect(queryByText('Disposition File')).toBeNull();
+      expect(queryByText('Lease/Licence File')).toBeNull();
```

Comment:
question - according to the test title this unit test is to show all create file options for non-retired properties but here we are asserting that the create Acquisition File is not present. 
Is this intended?